docs/contribution_guide.md

```diff
@@ -0,0 +1,79 @@
+# **Contributing Guide**
```

Comment:
@moses-kisakye could we move this guide to root of the project for now

Reply:
> @moses-kisakye could we move this guide to root of the project for now

the vision is to help new contributors easily onboard with contribution....

Reply:
you may take inspiration from [this](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md)

Reply:
> @HassanBahati 

Please Checkout #https://github.com/OSCA-Kampala-Chapter/autobot/pull/114/commits/a638ab2dd9c98f04dd24ec5fcfc79fe22f93744a though am looking forward to making it more of like [this](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md)

Reply:
Not really, we can have a README.md file, you may maintain the current. You may go ahead and create a file CONTRIBUTING.md and add the guide you've written there

Reply:
Ooooh, I get it know. Let me resolve it

Reply:
> @HassanBahati just tag me when this PR is ready for merging

sure

Reply:
alright